templates/cassiopeia/html/layouts/chromes/card.php

```diff
@@ -21,34 +21,19 @@
 }
 
 $moduleTag              = $params->get('module_tag', 'div');
-$moduleAttribs          = [];
+$modulePos              = $module->position;
 $moduleAttribs['class'] = $module->position . ' card ' . htmlspecialchars($params->get('moduleclass_sfx'), ENT_QUOTES, 'UTF-8');
-$headerTag              = htmlspecialchars($params->get('header_tag', 'h4'), ENT_QUOTES, 'UTF-8');
+$modId                  = 'mod-' . $module->id;
+$headerTag              = htmlspecialchars($params->get('header_tag', ''), ENT_QUOTES, 'UTF-8');
```

Comment:
To solve the issues above.

```suggestion
$headerTag              = htmlspecialchars($params->get('header_tag', 'h3'), ENT_QUOTES, 'UTF-8');
```

Reply:
> 
> 
> To solve the issues above.

I don't understand what the 'h3' in line 27 is for. The header tag is chosen in the individual module setup by selecting the header tag in the Advanced tab.

However, I don't see any of the rogue code in Firefox inspector (Windows 10). I have just updated to last night's nightly in case some code had changed, but it is the same.

This is a view on the front end (I have selected the h6 header tag)

![1-card-with-and-without-header](https://user-images.githubusercontent.com/36987494/94278226-7b503580-ff42-11ea-9367-2c9b52154077.jpg)


And this is the inspected code as expected:

![2-card-html](https://user-images.githubusercontent.com/36987494/94278248-81dead00-ff42-11ea-9104-8970ebbe69fd.jpg)

Reply:
This is for getting a default value when no value is set. It's not an issue in your case because you have already saved the module. But it is an issue with modules installed by default which don't have the parameter stored in the database.

Reply:
Ah. OK.